lib/howl/interactions/line_selection.moon

```diff
@@ -97,6 +97,10 @@ interact.register
     opts.matcher = matcher
     opts.on_change = line_match_highlighter(editor, matcher.explain)
     opts.force_preview = true
+    opts.columns = {
+        {align: 'right', style: 'comment', min_width: 3},
```

Comment:
That would require scanning the entire list of lines up ahead - maybe not a big deal for most cases but could occasionally be pretty large.